packages/yourloops/components/chat/chat-widget.tsx

```diff
@@ -179,113 +156,115 @@ function ChatWidget(props: ChatWidgetProps): JSX.Element {
     setInputText(inputText + emoji.emoji)
   }
 
-  const resetInputSize = (): void => {
-    if (!content.current || !inputRow.current) {
-      throw new Error('Cannot find elements for resize')
-    }
-    content.current.style.height = '290px'
-    inputRow.current.style.height = '60px'
-  }
-
   const sendMessage = async (): Promise<void> => {
     await ChatApi.sendChatMessage(teamId, patient.userid, inputText, privateMessage)
     const messages = await ChatApi.getChatMessages(teamId, patient.userid)
     setMessages(messages)
     setInputText('')
-    resetInputSize()
   }
 
-  const inputHandler = (event: React.ChangeEvent<HTMLTextAreaElement>): void => {
-    setInputText(event.target.value)
-    if (!content.current || !inputRow.current) {
-      throw new Error('Cannot find elements for resize')
-    }
-    // values for scrollHeight are 24 one line, 44 2 lines and 64 3 lines
-    // we display the scrollbar only when 3 lines are present
-    if (event.target.scrollHeight > 44) {
-      content.current.style.height = '250px'
-      inputRow.current.style.height = '100px'
-    } else if (event.target.scrollHeight > 24) {
-      content.current.style.height = '270px'
-      inputRow.current.style.height = '80px'
-    } else {
-      resetInputSize()
+  const onEmojiPickerKeyPress = (event: KeyboardEvent): void => {
+    if (event.key === 'Escape') {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Any idea of if it is easy to close the emoji stuff on mobile?

Reply:
"We are not responsive" says the manual 😅

Reply:
I think it will be complicated to do this on mobile, the library doesn't seem to bo compatible with mobile. Maybe with the back arrow on android but don't know if we can access to the phone API easily with the app. Maybe if we make a PWA, but not on this PR ^^